src/wp-includes/class-wp-theme-json.php

```diff
@@ -2736,6 +2765,20 @@ protected static function remove_insecure_settings( $input ) {
 				}
 			}
 		}
+
+		// Ensure indirect properties not included in any `PRESETS_METADATA` value are allowed.
+		foreach ( static::INDIRECT_PROPERTIES_METADATA as $property => $paths ) {
+			foreach ( $paths as $path ) {
+				$value = _wp_array_get( $input, $path, array() );
```

Comment:
Why a default of `array()`? The code isn't expecting or processing an array. I'd recommend removing it to let `_wp_array_get()` return `null` as the default. 

```suggestion
				$value = _wp_array_get( $input, $path );
```

Reply:
https://github.com/WordPress/wordpress-develop/pull/3742/commits/c05ca999b5875bb06eda819ce723d3adb1b42307